src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1703,7 +1703,16 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
                         positionGetFunc: () => 0,
                         canReadFunc: () => true,
                         readAsyncFunc: (buffer, offset, count, cancellationToken) => syncFailure ? throw error : Task.Delay(1).ContinueWith<int>(_ => throw error)));
-                    Assert.Same(error, await Assert.ThrowsAsync<FormatException>(() => client.PostAsync(uri, content)));
+
+                    if (!UseManagedHandler && !PlatformDetection.IsUap)
```

Comment:
How did the test pass on UWP prior to your change? Or was it always failing?

Reply:
The test is outerloop.  I didn't run outerloop tests on UAP.

Reply:
This tests are in outer loop so my assumption is that Steve ran outer loop locally but not for UWP. 

As for the managed handler: at the time of his local run it was passing with the original exception but the exception changed after integration with other recent changes to ManagedHandler.

Reply:
OK, that makes sense - we probably didn't look recently on UWP nightly runs where the failure would have showed up as well.